src/foam/nanos/cron/SimpleIntervalScheduleView.js

```diff
@@ -43,9 +43,9 @@ foam.CLASS({
             })
           }
         ).addClass(this.myClass('container'))
-        .start(data.START_DATE.__).addClass(this.myClass('fullWidth')).end()
-        .start(data.REPEAT.__).addClass(this.myClass('halfWidth')).end()
+        .start(data.START_DATE.__).addClass(this.myClass('halfWidth')).end()
```

Comment:
Same here, this would impact the layout of schedules everywhere so why are we changing this behaviour?